kornia/geometry/boxes.py

```diff
@@ -341,25 +433,33 @@ def transform_boxes_(self, M: torch.Tensor) -> "Boxes":
         return self.transform_boxes(M, inplace=True)
 
     @property
-    def _is_batch(self) -> bool:
-        return self._boxes.ndim == 4
+    def _is_batched(self) -> bool:
```

Comment:
```suggestion
    def is_batched(self) -> bool:
```
I think properties should be public, we can do in later pr